src/LegacyPortal.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
It might be worth adding a comment at the top of this file explaining that this is used by React 15, or something like that.